rules/PHPUnit90/Rector/MethodCall/UseOnlyMethodRector.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PHPUnit\PHPUnit90\Rector\MethodCall;
```

Comment:
is it phpunit 9 or phpunit 10? If phpunit 10, then it needs in `PHPUnit100` namespace imo:

it also may need to be added to the phpunit 10 setlist

https://github.com/rectorphp/rector-phpunit/blob/main/config/sets/phpunit100.php

Reply:
it was deprecated phpuni9 and removed phpunit10 in this case do we apply for versions that we are removing?

Reply:
I see, but it seems still need to be registered to `config/sets/phpunit100.php`

Reply:
then should I move to PhpUnit100 folder and register in `config/sets/phpunit100.php`?

Reply:
I think using `PHPUnit` directory is better, but probably @TomasVotruba has opinion?

Reply:
Hey @samsonasik 

I guess I could just use 

it 😄 
```php
new MethodCallRename('PHPUnit\Framework\Assert', 'setMethods', 'onlyMethods'),
```

Reply:
> https://github.com/rectorphp/rector-phpunit/blob/main/config/sets/phpunit100.php

like this https://github.com/rectorphp/rector-phpunit/blob/main/config/sets/phpunit100.php#L58 😄

Reply:
yeah, that seems good 👍